releases/release-1.31/release-notes/maps/pr-124884-map.yaml

```
@@ -0,0 +1,5 @@
+pr: 124884
+releasenote:
+  text: Added a feature to report an event to pod if kubelet does `attach operation
+    failed` when kubelet is running with `--enable-controller-attach-detach=false`.
```

Is this a useful summary:
```suggestion
  text: Added a feature to report an event about a Pod if kubelet observes a failed attach
     operation, even if the kubelet is running with `--enable-controller-attach-detach=false`.
```
?

---

yes I like it much better